charts/rancher-turtles/templates/post-upgrade-job.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if and (eq (index .Values "rancherTurtles" "features" "managementv3-cluster" "enabled") true) (eq (index .Values "rancherTurtles" "features" "managementv3-cluster-migration" "enabled") true) }}
```

Comment:
why we do not need to enable `managementv3-cluster` feature flag anymore?

Reply:
Downgrade procedure involves `managementv3-cluster` being disabled. The desired behavior was explained in ADR.